OpenMEEG/wrapping/src/test_python.py

```diff
@@ -1,4 +1,20 @@
 #!/usr/bin/env python
+###########################################
+# CAVEAT on using OpenMEEG from python !!
+# Beware that a temporary object has its memory
+# released. So do not work with data provided from
+# an OpenMEEG temporary object.
+# For example, having a symmetric matrix defined as :
+# > M = om.SymMatrix(100)
+# taking as an numpy array the sub-matrix of M might lead to corrupted memory:
+# > mySubMat = om.asarray(M.submat(0,10,0,10))
+# since submat returns a newly created object that is hold by nothing, thus
+# destroyed afterward.
+# Instead do keep an object pointing the newly created submatrix, and then
+# access the numpy array form of it:
+# > subM = M.submat(0,10,0,10)
+# > mySubMat = om.asarray(subM)
+###########################################
```

Comment:
this message should be put in:

https://raw.githubusercontent.com/openmeeg/openmeeg_sample_data/master/compute_leadfields.py

thanks a lot